index.html

```diff
@@ -14,6 +14,7 @@ <h1>GitHUb Branch, Merge</h1>
         <ul>
             <li>Home</li>
             <li>Contact</li>
+            <li>About</li>

```

Comment:
change About to Services

Reply:
@IdrissaMurenga could you take this and work on it as @Batmanwilly2022 is out today?

Reply:
@Yunusib12 All fixed